autoload.php

```diff
@@ -9,48 +9,36 @@
  * file that was distributed with this source code.
  */
 
-use bitExpert\PHPStan\Magento\Autoload\Cache\FileCacheStorage;
 use bitExpert\PHPStan\Magento\Autoload\FactoryAutoloader;
 use bitExpert\PHPStan\Magento\Autoload\MockAutoloader;
 use bitExpert\PHPStan\Magento\Autoload\ProxyAutoloader;
 use bitExpert\PHPStan\Magento\Autoload\TestFrameworkAutoloader;
-use Nette\Neon\Neon;
 use PHPStan\Cache\Cache;
+use PHPStan\DependencyInjection\Container;
+
+if (!isset($container) || !$container instanceof Container) {
+    return;
```

Comment:
Early return in case the autoloader is included in some unexpected way. Should we exit with instructions?

Reply:
maybe echo'ing some warning might be a good idea. I mean technically the user might not be able to do something, as it's an internal problem of PHPStan, but at least it's clear then why the autoloaders not work.